include/morphio/mut/writers.h

```diff
@@ -6,6 +6,8 @@ namespace writer {
 void swc(const Morphology& morphology, const std::string& filename);
 void asc(const Morphology& morphology, const std::string& filename);
 void h5(const Morphology& morphology, const std::string& filename);
+void _asc_to_stream(const Morphology& morphology, std::ostream& stream);
```

Comment:
These two new functions are only used in the binding code. I don't know if I should keep the underscore or not.

Reply:
Can we put them in the bindings, so they don't become part of the `official` interface?